examples/automatic_speech_recognition/automatic_speech_recognition/utils.py

```diff
@@ -0,0 +1,87 @@
+import re
+import difflib
+from numwords_to_nums.numwords_to_nums import NumWordsToNum
+
+from typing import Union
+
+from workflow import GroundTruth
+from workflow import Inference
+
+def generate_diff_word_level(reference: str, candidate: str, mode: str):
+        '''
+        Calculates the diff between the reference and candidate texts, and returns the following:
+            output: text wrapped with false postive / false negative tags
```

Comment:
Should we use an object here so we don't have to deal with a long list of returned items?  It will be nice to clean this up a bit more.